cypress/e2e/createVault.cy.ts

```diff
@@ -1,102 +0,0 @@
-import { deriveAccount, gqlQuery, injectWeb3 } from '../util';
```

Comment:
cypress is currently disabled - do you wan tto try to reenable it?

Reply:
@crabsinger  ok I will try it